api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentGroupController.java

```diff
@@ -332,6 +361,180 @@ private Payment getPayment(PaymentFeeLink paymentFeeLink, String paymentReferenc
             .orElseThrow(() -> new PaymentNotFoundException("Payment with reference " + paymentReference + " does not exists."));
     }
 
+    @ApiOperation(value = "Record a Bulk Scan Payment", notes = "Record a Bulk Scan Payment")
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = "Bulk Scan Payment created"),
+        @ApiResponse(code = 400, message = "Bulk Scan Payment creation failed"),
+        @ApiResponse(code = 422, message = "Invalid or missing attribute")
+    })
+    @PostMapping(value = "/payment-groups/{payment-group-reference}/bulk-scan-payments-strategic")
+    @ResponseBody
+    @Transactional
+    public ResponseEntity<PaymentDto> recordBulkScanPaymentStrategic(@PathVariable("payment-group-reference") String paymentGroupReference,
+                                                                     @Valid @RequestBody BulkScanPaymentRequestStrategic bulkScanPaymentRequestStrategic,
+                                                                     @RequestHeader(required = false) MultiValueMap<String, String> headers) throws CheckDigitException {
+        // Check Any Duplicate payments for current DCN
+        if (bulkScanPaymentRequestStrategic.getDocumentControlNumber() != null) {
```

Comment:
I believe we already have Duplicate DCN validation exists in Bulk Scan System, seems like we are repeating the same in down-stream systems.

Reply:
This validation is from payhub side to avoid duplication of req, which is one of the sole purposes of this Jira

https://tools.hmcts.net/jira/browse/PAY-3147 for more details